sip-ri/src/main/java/gov/nist/javax/sip/parser/extensions/JoinParser.java

```diff
@@ -21,7 +21,7 @@ public class JoinParser extends ParametersParser {
     /**
      * Creates new CallIDParser
      * @param callID message to parse
-     */
+     */ 
```

Comment:
can we avoid adding those  one char changes to the commit, it makes the review a bit painful

Reply:
I am so sorry for inconvenience, I cleaned by git status odd changes, I guess it was added earlier, before I acknowledged this problem of mine. Will not repeat this mistake. My apologies